distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -1,3 +1,232 @@
+The *Details* tab provides a list of specific metadata.
+
+<<_viewing_revision_history,History>>:: View revision history of this resource.
+<<_editing_associations_on_a_record,Associations>>:: View or edit the relationship(s) between this resource and others in the catalog.
+<<_viewing_metadata_quality,Quality>>:: View the completeness and accuracy of the metadata for this resource.
+<<_exporting_a_record,Actions>>:: Export the metadata/resource to a specific format.
+<<_archiving_a_record,Archive>>:: Remove the selected record from standard search results.
+<<_overwriting_a_record,Overwrite>>:: Overwrite a record.
+
+==== Editing Records
+
+Records can be edited from the *Summary* or *Details* tabs.
+
+==== Editing Associations on a Record
+
+Update relationships between records through *Associations*.
+
+. Select the desired record from the *Results* tab.
+. Select the *Associations* tab.
+. Select *Edit*.
+. For a new association, select *Add Association*. (Only items in the current result set can be added as associations).
+.. Select the related record from either the *Parent* or *Child* menu.
+.. Select the type of relationship from the *Relationship* menu.
+.. Select *Save*.
+. To edit an existing association, update the selections from the appropriate menu and select *Save*.
+
+==== Viewing Revision History
+
+View the complete revision history of a record.
+
+. Select the desired record from the *Results* tab.
+. Select the *History* tab.
+.. Select a previous version from the list to preview.
+.. If desired, select *Revert to Selected Version* to undo changes made after that revision.
+
+==== Viewing Metadata Quality
+
+View and fix issues with metadata quality in a record.
+
+[NOTE]
+====
+Correcting metadata issues may require heightened permissions.
+====
+
+. Select the desired record from the *Results* tab.
+. Select *More*.
+. Select *Quality*.
+. A report is displayed showing any issues:
+.. Metacard Validation Issues.
+.. Attribute Validation Issues.
+
+==== Exporting a Record
+
+Export and view a resource and/or its metadata into another format.
+
+. Select the desired record from the *Results* tab.
+. Select *More*.
+. Select *Actions*.
+. Available export formats are listed. Select desired format.
+. Export opens in a new browser tab. Save, if desired.
+
+==== Archiving a Record
+
+To remove a record from active search results, archive it.
+
+. Select the desired record from the *Results* tab.
+. Select *More*.
+. Select *Archive*.
+. Confirm archiving the record.
+
+==== Restoring an Archived Record
+
+Restore an archived record to return it to active search results.
+
+. Select the *Search Archived* option from the query options menu.
+. Select the desired record from the *Results* tab.
+. Select *Archive*.
+. Confirm restoring the record.
+
+==== Overwriting a Record
+
+Replace a resource.
+
+. Select the desired record from the *Results* tab.
+. Select *More*.
+. Select *Overwrite*.
+
+
```

Comment:
Missing section: Recommend adding details about the `Notifications` area of the UI, denoted by the bell icon. 
![screenshot - notifications](https://cloud.githubusercontent.com/assets/16792154/21824394/786b48b4-d73c-11e6-9aab-2bbe9d536151.png)